### DISCUSSION TITLE
使用x64dbg调试微信老是退出，有什么办法绕过微信的反调试机制吗？

### DISCUSSION THREAD

**Discussion by ldtech007** [ttttupup/wxhelper#438] [General]
请问使用x64dbg调试微信老是退出，有什么办法绕过微信的反调试机制吗？

**Comment by ttttupup** [comment]
没有反调试吧，没遇到过

**Reply by lxy2500798479** [reply]
我一附加 微信就卡住了 这是有反调试吗

**Comment by ttttupup** [comment]
如果遇到了，可以试试https://github.com/x64dbg/ScyllaHide  反调试的插件

**Comment by lxy2500798479** [comment]
> 请问使用x64dbg调试微信老是退出，有什么办法绕过微信的反调试机制吗？

大佬解决了吗
